frontend/app/components/validation-date-input.js

```diff
@@ -0,0 +1,11 @@
+import Component from "@ember/component";
+
+export default Component.extend({
+  actions: {
+    setBirthdate(value) {
+      this.get("setDirty")();
+      this.get("update")(value);
```

Comment:
I have never seen something like this with two parantheses. Aren't there any better ways?

Reply:
I didn't find any other ways, so probably no.